src/main/java/com/google/alpollo/YouTubeServlet.java

```diff
@@ -0,0 +1,48 @@
+package com.google.alpollo;
+
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.model.SearchListResponse;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/youtube")
+public class YouTubeServlet extends HttpServlet {
+  /**
+   * Accesses YouTube and retrieves a collection of videos relating to the given query/words found
+   * in the request. Returns only the snippet for each video and returns only videos, no playlists
+   * or channels allowed.
+   */
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Separating the code inside `doGet()` into smaller code-blocks with whitespaces might make the code easier to read.